src/renderers/client/render.tsx

```
@@ -0,0 +1,289 @@
+        ) : (
+            <div className={'svgContainerStyle'} dangerouslySetInnerHTML={{ __html: data.toString() }} />
+        );
+        return (
+            <div
+                className={'display'}
+                style={divStyle}
+                onMouseOver={onMouseOver}
+                onMouseOut={onMouseOut}
+                onFocus={onMouseOver}
+                onBlur={onMouseOut}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)